apiml-common/src/main/java/org/zowe/apiml/product/web/HttpConfig.java

```diff
@@ -157,10 +163,10 @@ public void init() {
 
         } catch (HttpsConfigError e) {
             log.error("Invalid configuration of HTTPs: {}", e.getMessage());
-            System.exit(1);
+            throw e;
```

Comment:
I don't think it is the best approach to log and rethrow. If it's not possible to recover from the exception, the application should be killed. The problem seems to be with the process of how it is being killed.

Reply:
@achmelo I am open to suggestions. I was just reading on the internet and I could only find suggestions to throw an exception. I initially tried to throw a `RuntimeException` but then Sonar complained that throwing generic exceptions was not good. Another suggestion I found was `Runtime.getRuntime().halt()` but it sounded even more intrusive.